bindings_ffi/src/mls.rs

```diff
@@ -2272,8 +2272,8 @@ impl FfiConversation {
         self.inner.added_by_inbox_id().map_err(Into::into)
     }
 
-    pub async fn group_metadata(&self) -> Result<Arc<FfiConversationMetadata>, GenericError> {
-        let metadata = self.inner.metadata().await?;
+    pub fn group_metadata(&self) -> Result<Arc<FfiConversationMetadata>, GenericError> {
+        let metadata = self.inner.metadata()?;
```

Comment:
@mchenani mind gut checking this. I know you were the one who made metadata async awhile back maybe for forking?

Reply:
The reason was we didn't have groups sync lock, since now we have, that would be ok.